lib/commands/validate-traffic.ts

```diff
@@ -0,0 +1,23 @@
+import * as yargs from "yargs"
+
+import { cliSuppressExceptions } from "../cliSuppressExceptions"
+import { log } from "../util/logging"
+import * as validate from "../validate"
+
+export const command = "validate-traffic <traffic-path> <spec-path>"
+export const describe = "Validate the spec file based on the traffic payload"
+
+export async function handler(argv: yargs.Arguments): Promise<void> {
+  await cliSuppressExceptions(async () => {
+    log.debug(argv.toString())
+    const specPath = argv.specPath
+    const trafficPath = argv.trafficPath
+    const vOptions: validate.Options = {
+      consoleLogLevel: argv.logLevel,
+      logFilepath: argv.f,
+      pretty: argv.p
```

Comment:
pretty option seems not implemented yet?

Reply:
Others command also add pretty option.

Reply:
Without pretty option output:

![image](https://user-images.githubusercontent.com/46650314/123891601-bb3dc080-d98b-11eb-8f8c-84337ed18f15.png)


With pretty option output: 

![image](https://user-images.githubusercontent.com/46650314/123891462-8893c800-d98b-11eb-8de3-d2c415fc3c79.png)